Index: vendor/magento/module-catalog-inventory/Model/ResourceModel/Indexer/Stock/DefaultStock.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- vendor/magento/module-catalog-inventory/Model/ResourceModel/Indexer/Stock/DefaultStock.php	(revision )
+++ vendor/magento/module-catalog-inventory/Model/ResourceModel/Indexer/Stock/DefaultStock.php	(revision )
@@ -300,7 +300,7 @@
             $data[] = [
                 'product_id' => (int)$row['entity_id'],
                 'website_id' => (int)$row['website_id'],
-                'stock_id' => Stock::DEFAULT_STOCK_ID,
+                'stock_id' => (int)$row['stock_id'],
                 'qty' => (double)$row['qty'],
                 'stock_status' => (int)$row['status'],
             ];